### PR TITLE
Add a basic "how to add vouchers" page to the administration section

### PR DESCRIPTION
**Pull request by drewmelck**

### CODE REVIEW

File: src/administration/vouchers.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Vouchers 
+
+## Applying a voucher to your project
+
+If you receive a [Platform.sh](https://platform.sh/) voucher you can redeem it as follows:
```
Comment (Crell) [inline src/administration/vouchers.md]: Our name in the Docs shouldn't link to the site.  It's already known and linked from a zillion places.

File: src/administration/vouchers.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+* Go to your account settings via the UI link in the top-right corner of the screen (where your profile picture is) or using this link: [https://accounts.platform.sh/user](https://accounts.platform.sh/user).
+* In the left navigation click on vouchers (the seventh entry)
+* On the page click on "add a voucher code"
+* Enter the code and click on the "ADD CODE" button
+* Et voila! Your account will now be credited with additional dollar, pounds or euros. 
+
+If you are assessing Platform.sh for your organziation and think that you could benefit from a little more oomf in your test project why not contact us to request a voucher? Write to: sales@platform.sh.
```
Comment (Crell) [inline src/administration/vouchers.md]: Should this be encoded somehow so that it's not quite so easy to spam harvest?  Or is that already a lost battle?

**Comment by Crell** [comment]
This also needs to be added to SUMMARY.md so that it shows up in the menu.

File: src/administration/vouchers.md (hunk 2 above)
Comment (pjcdawkins) [inline src/administration/vouchers.md]: Typo in 'organization' on the same line

File: src/administration/vouchers.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+If you receive a Platform.sh voucher code you can redeem it as follows:
+
+* Go to your account settings via the UI link in the top-right corner of the screen (where your profile picture is) or using this link: [https://accounts.platform.sh/user](https://accounts.platform.sh/user).
+* In the left navigation click on vouchers (the seventh entry)
+* On the page click on "add a voucher code"
+* Enter the code and click on the "ADD CODE" button
+* Et voila! Your account will now be credited with additional dollar, pounds or euros. 
+
+If you are assessing Platform.sh for your organization and think that you could benefit from a little more oomf in your test project why not contact us to request a voucher? You can tell us more at: [https://platform.sh/contact/](https://platform.sh/contact/)
```
Comment (mrkschan) [inline src/administration/vouchers.md]: what does `oomf` mean here??

Comment (pjcdawkins) [inline src/administration/vouchers.md]: Out Of Memory Forbearance

**Pull request merged**